custom/icds_reports/reports/cas_reach_data.py

```diff
@@ -91,62 +104,83 @@ def get_data_for_daily_usage(date, filters):
-                    'percent': None,
-                    'value': get_value(awc_this_month_data, 'states'),
-                    'all': None,
-                    'format': 'number',
-                    'frequency': 'month',
-                }
-            ]
+    cas_reach_records = [
+        [
+            {
+                'label': _('AWCs Launched'),
+                'help_text': awcs_launched_help_text(),
+                'color': None,
+                'percent': None,
+                'value': awcs,
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+                'redirect': 'icds_cas_reach/awcs_covered'
+            },
+            number_of_awc_open_yesterday
+        ],
+        [
+            {
+                'label': _('Sectors covered'),
+                'help_text': _('Total Sectors that have launched ICDS CAS'),
+                'percent': None,
+                'value': get_value(awc_this_month_data, 'supervisors'),
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+            },
+            {
+                'label': _('Blocks covered'),
+                'help_text': _('Total Blocks that have launched ICDS CAS'),
+                'percent': None,
+                'value': get_value(awc_this_month_data, 'blocks'),
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+            },
+        ],
+        [
+
+            {
+                'label': _('Districts covered'),
+                'help_text': _('Total Districts that have launched ICDS CAS'),
+                'percent': None,
+                'value': get_value(awc_this_month_data, 'districts'),
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+            },
+            {
+                'label': _('States/UTs covered'),
+                'help_text': _('Total States that have launched ICDS CAS'),
+                'percent': None,
+                'value': get_value(awc_this_month_data, 'states'),
+                'all': None,
+                'format': 'number',
+                'frequency': 'month',
+            }
         ]
+    ]
+
+    display_lss_launched = current_month > datetime(2020, 1, 1) and show_prerelease_features
+
+    if display_lss_launched:
+        ls_launched_data = get_data_for_ls_launched(current_month, config)
+        number_of_lss_launched = {
+            'label': _('LSs Launched'),
+            'help_text': ls_launched_help_text(),
+            'color': None,
+            'percent': None,
+            'value': get_value(ls_launched_data, 'ls_launched'),
+            'all': None,
+            'format': 'number',
+            'frequency': 'month',
+            'redirect': 'icds_cas_reach/ls_launched'
+        },
+        cas_reach_records.insert(1, number_of_lss_launched)
```

Comment:
this feels like inserting like this will display it in wrong fashion.

Reply:
Updated this